<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_window</class>
 <widget class="QDialog" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(240,240,240); color: rgb(85,87,83);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background: rgb(240,240,240); border: none; border-radius: 40px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85,87,83); background: transparent;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="settings_frame">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#settings_frame {
border: 1px solid rgb(220,220,220);
color: rgb(85,87,83);
border-radius: 40px;
background: white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,0,1">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>30</number>
         </property>
         <item row="3" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="time_mode_label">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>Admin Mode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="admin_mode_switch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider { background: transparent; }

QSlider::groove:horizontal {
border: 1px solid rgb(220,220,220);
background: rgb(160, 160, 160);
height: 20px;
border-radius: 10px;
}

QSlider::handle:horizontal {
height: 20px;
width: 20px;
border: none;
border-radius: 10px;
background: rgb(85,87,83);
}

QSlider::handle:horizontal:hover {
background: rgb(66,68,65);
}

QSlider::groove:horizontal:disabled {
background: white;
}

QSlider::handle:horizontal:disabled {
background: rgb(220,220,220);
}</string>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="time_mode_switch">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider { background: transparent; }

QSlider::groove:horizontal {
border: 1px solid rgb(220,220,220);
background: rgb(160, 160, 160);
height: 20px;
border-radius: 10px;
}

QSlider::handle:horizontal {
height: 20px;
width: 20px;
border: none;
border-radius: 10px;
background: rgb(85,87,83);
}

QSlider::handle:horizontal:hover {
background: rgb(66,68,65);
}

QSlider::groove:horizontal:disabled {
background: white;
}

QSlider::handle:horizontal:disabled {
background: rgb(220,220,220);
}</string>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="admin_mode_label">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="blink_switch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider { background: transparent; }

QSlider::groove:horizontal {
border: 1px solid rgb(220,220,220);
background: rgb(160, 160, 160);
height: 20px;
border-radius: 10px;
}

QSlider::handle:horizontal {
height: 20px;
width: 20px;
border: none;
border-radius: 10px;
background: rgb(85,87,83);
}

QSlider::handle:horizontal:hover {
background: rgb(66,68,65);
}

QSlider::groove:horizontal:disabled {
background: white;
}

QSlider::handle:horizontal:disabled {
background: rgb(220,220,220);
}</string>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>24 hr time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="blink_label">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85,87,83); background: transparent;</string>
           </property>
           <property name="text">
            <string>Blink for Errors</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="done_button">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>52</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover:pressed {
background-color: rgb(48,49,47);
}

QPushButton {
	background-color: rgb(85,87,83);
	border: none;
	border-radius: 26px;
	color: white;
	padding-left: 6px;
}

QPushButton:hover {
background-color: rgb(66,68,65);
}

QPushButton:disabled {
	background-color: rgb(215, 215, 215);
}</string>
        </property>
        <property name="text">
         <string>DONE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
